jdl/validators/jdl-without-application-validator.js

```diff
@@ -46,13 +46,22 @@ module.exports = {
  * @param {String} applicationSettings.applicationType - the application type.
  * @param {String} applicationSettings.databaseType - the DB type.
  * @param {Boolean} applicationSettings.skippedUserManagement - whether user management is skipped.
+ * @param {Object} applicationSettings.blueprints - the blueprints used.
  * @param {Object} logger - the logger to use, default to the console.
  */
 function createValidator(jdlObject, applicationSettings = {}, logger = console) {
     if (!jdlObject) {
         throw new Error('A JDL object must be passed to check for business errors.');
     }
 
+    if (applicationSettings.blueprints && applicationSettings.blueprints.length !== 0) {
+        return {
+            checkForErrors: () => {
+                logger.warn('Generating application with blueprint, skipping jdl validation');
```

Comment:
This message implies the validator knows about the context where it's being used, meaning the validator knows we're using it to generate applications. It shouldn't.
What about something like "Blueprints are being used, the JDL validation phase is skipped."?

Reply:
ok with this